src/_includes/install/sampledata/sample-data-clone.md

```diff
@@ -87,6 +87,10 @@ To clone the Magento sample data GitHub repository using the SSH protocol:
 
 1. See [Set file system permissions and ownership](#samp-data-perms).
 
+1. ```bash
```

Comment:
The same here.

Thank you!

Reply:
@atwixfirster 
Thanks for your notes.

Everything is done at your request.
Could you please proceed with the code review?